src/expiration-year.js

```diff
@@ -10,9 +10,11 @@ function verification(isValid, isPotentiallyValid, isCurrentYear) {
   };
 }
 
-function expirationYear(value) {
+function expirationYear(value, maxElapsedYear) {
```

Comment:
I renamed maxYear to maxElapsedYear because maxYear might remind someone of the dominical year.